rollup.config.js

```diff
@@ -1,3 +1,5 @@
+const dotenv = require("dotenv");
```

Comment:
Is that necessary? I don't see any code that relies on the dotenv later on...

Reply:
Where we inject the signal / api server addresses on line 42+, right?